test/e2e/serial/tests/non_regression.go

```diff
@@ -366,6 +366,9 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			failedPodIds := e2efixture.WaitForPaddingPodsRunning(fxt, paddingPods)
 			Expect(failedPodIds).To(BeEmpty(), "some padding pods have failed to run")
 
+			By("waiting for the NRT data to settle")
+			e2efixture.WaitForNRTSettle(fxt)
```

Comment:
let's use brand new `MustSettleNRT` (please rebase on top of current head of main)

Reply:
done, thanks.